docs/decisions/0016_automatic_expiration.rst

```diff
@@ -0,0 +1,55 @@
+0016 Automatic Cancellation (Expiration)
+****************************************
+
+Status
+======
+Accepted - January 2024
+
+Context
+=======
+
+How expiration currently works
+------------------------------
+There's currently a management command (run via a cron) named ``automatically_exire_assignments.py``
+that toggles the state of ``LearnerContentAssignment`` records ("Assignments")
+to ``CANCELLED`` under any of the following conditions:
+
+1. The current state of the assignment is ``ALLOCATED``.
+2. The current date is more than 90 days after the most recent notification action - this action
+   time is used as a proxy for understanding *when* the assignment *was last allocated*. This helps
+   deal with the edge case where an assignment is allocated, cancelled, and re-allocated later. Note
+   that a reminder action on an assignment record *does not* reset this specific expiry time.
+3. The current date is greater than the inferred enrollment deadline for the assigned course.
+4. The current date is greater than the expiration date of the subsidy associated with the Assignment's
+   access policy record.
```

Comment:
Nit: these list elements can't be ORed together as the previous paragraph suggests, it's more like 1 AND (2 OR 3 OR 4).  Maybe move 1 out of the list?